tests/manage/test_pvc_disruptive.py

```diff
@@ -21,43 +61,171 @@ class BaseDisruption(ManageTest):
     pvc_obj = None
     storage_type = None
 
+    def create_pvcs(self):
+        """
+        Create PVCs
+
+        Returns:
+            list: List of PVC objects
+        """
+        # Parameters for PVC yaml as dict
+        pvc_data = load_yaml_to_dict(constants.CSI_PVC_YAML)
+        pvc_data['metadata']['namespace'] = self.namespace
+        pvc_data['spec']['storageClassName'] = self.sc_obj.name
+        pvc_data['metadata']['name'] = helpers.create_unique_resource_name('test', 'pvc')
+
+        # Create 5 PVCs
+        pvc_objs = create_multiple_pvc(5, pvc_data)
+        return pvc_objs
+
+    def create_pods(self, pvc_objs_list):
+        """
+        Create Pods.
+        A pod will be created for each PVC in 'pvc_objs_list'.
+
+        Args:
+            pvc_objs_list (list): List of PVC objects
+
+        Returns:
+            list: List of Pod objects
+        """
+        pod_objs = []
+        for pvc_obj in pvc_objs_list:
+            pod_obj = helpers.create_pod(
+                interface_type=constants.CEPHBLOCKPOOL, pvc_name=pvc_obj.name, wait=False, namespace=self.namespace
+            )
+            pod_objs.append(pod_obj)
+        return pod_objs
+
     def disruptive_base(self, operation_to_disrupt, resource_to_delete):
         """
         Base function for PVC disruptive tests
+        Deletion of 'resource_to_delete' will be introduced while 'operation_to_disrupt' is progressing
         """
+        # Fetch the number of Pods and PVCs
+        initial_num_of_pods = len(pod.get_all_pods(namespace=self.namespace))
```

Comment:
['items'] ?

Reply:
get_all_pods is returning a list of PVC objects. Number of elements in the list will be equal to number of PVC